app/models/school.rb

```diff
@@ -21,14 +21,11 @@ def self.seed_somerville_schools
     School.create([
       { state_id: 15, local_id: "BRN", name: "Benjamin G Brown", school_type: "ES" },
       { state_id: 75, local_id: "HEA", name: "Arthur D Healey", school_type: "ESMS" },
-      { state_id: 83, local_id: "KDY", name: "John F Kennedy", school_type: "ESMS" },
       { state_id: 87, local_id: "AFAS", name: "Albert F. Argenziano School", school_type: "ESMS" },
       { state_id: 111, local_id: "ESCS", name: "E Somerville Community", school_type: "ESMS" },
       { state_id: 115, local_id: "WSNS", name: "West Somerville Neighborhood", school_type: "ESMS" },
       { state_id: 120, local_id: "WHCS", name: "Winter Hill Community", school_type: "ESMS" },
-      { state_id: 410, local_id: "NW", name: "Next Wave Junior High", school_type: "MS" },
       { state_id: 505, local_id: "SHS", name: "Somerville High", school_type: "HS" },
-      { state_id: 510, local_id: "FC", name: "Full Circle High School", school_type: "HS" }
```

Comment:
From your notes, it sounds like this didn't reduce the record count, could we put these lines back in?

Reply:
OK, would you like just FC school to come back, or all 3 schools?

Reply:
Let's bring 'em all back!